src/projection/animation/mix-values.ts

```diff
@@ -47,17 +51,13 @@ export function mixValues(
         followRadius ||= 0
         leadRadius ||= 0
 
-        /**
-         * Currently we're only crossfading between numerical border radius.
-         * It would be possible to crossfade between percentages for a little
-         * extra bundle size.
-         */
-        if (
-            typeof followRadius === "number" &&
-            typeof leadRadius === "number"
-        ) {
-            const radius = Math.max(mix(followRadius, leadRadius, progress), 0)
-            target[borderLabel] = radius
+        target[borderLabel] = Math.max(
+            mix(asNumber(followRadius), asNumber(leadRadius), progress),
+            0
+        )
+
+        if (percent.test(leadRadius) || percent.test(followRadius)) {
+            target[borderLabel] += "%"
```

Comment:
What happens if one is pixels and the other is percent? Then we would treat them both as percent, right?

Reply:
(Also don't see that in a test case)

Reply:
Yeah currently the crossfade breaks completely, in this case they'd be treated as a percentage. I'm not sure either is ideal though we don't generally support animating between value types anywhere else either.

Reply:
Wouldn't it be better to not animate then do something weird?

Reply:
Ah yeah I've just pushed this fix